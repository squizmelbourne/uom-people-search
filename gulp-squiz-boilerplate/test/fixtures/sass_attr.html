<!-- build:sass test/.tmp/styles/sass_attr.css {"media": "print" } -->
    <link rel="stylesheet" href="test/fixtures/styles/main.scss" />
<!-- endbuild -->