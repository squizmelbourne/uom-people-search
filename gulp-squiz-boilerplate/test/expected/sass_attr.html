<link href="test/.tmp/styles/sass_attr.css" rel="stylesheet" media="print" />